extensions/wikia/GlobalNavigation/scripts/GlobalNavigationSearch.js

```diff
@@ -20,6 +21,15 @@ require(['jquery', 'wikia.browserDetect', 'GlobalNavigationiOSScrollFix'], funct
 		} else {
 			setPropertiesOnInput(true);
 		}
+
+		//Disables search button until the user has entered at least one character.
+		$searchInput.keydown(function () {
```

Comment:
Move this to function which is triggered after DOM is loaded. On Special Pages all scripts are loaded in <head> and it may introduce race conditions.